vnext/Microsoft.ReactNative/ABICxxModule.h

```diff
@@ -45,4 +45,6 @@ struct ABICxxModule : facebook::xplat::module::CxxModule {
   std::vector<ConstantProvider> m_constants;
 };
 
+constexpr auto DefaultEventEmitterName = "RCTDeviceEventEmitter";
```

Comment:
>constexpr auto [](start = 0, length = 14)

Generally it is very inefficient in C++ to define constants in the header file.

Reply:
Fixed.